supervisor/homeassistant/websocket.py

```diff
@@ -21,8 +21,8 @@
 
 MIN_VERSION = {
     WSType.SUPERVISOR_EVENT: "2021.2.4",
-    WSType.BACKUP_START: "2021.12.0",
-    WSType.BACKUP_END: "2021.12.0",
+    WSType.BACKUP_START: "2022.2.0",
```

Comment:
Should this be `2022.1.0` so it includes dev/beta builds?

Reply:
Yeah I was actually wondering: So `2022.2.0.dev20220110` < `2022.2.0`?

We bumped already to 2022.2.0, and fix will only be in a 2022.2.0 dev build...

Reply:
`2022.2.0.dev20220110` is considered lower than `2022.2.0`

Reply:
Changed to `2022.1.0`.